2-mandatory/2-financial-times.js

```diff
@@ -5,16 +5,33 @@
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
 function potentialHeadlines(allArticleTitles) {
```

Comment:
This is a good solution!
For extra practice, you could try re-writing this with the `filter` array method.

Reply:
Trying other methods and ways of solving code is something I'm going to be aiming for. Thank you.